hat/assets/js/apps/Iaso/domains/users/components/UsersDialog.tsx

```diff
@@ -176,6 +183,17 @@ const UserDialogComponent: FunctionComponent<Props> = ({
             !allUserUserRolesPermissions.includes(Permissions.ORG_UNITS),
         );
     }, [allUserRolesPermissions.length, allUserUserRolesPermissions]);
+    const isNewUser = !user.id?.value;
+    const allowConfirm =
+        !hasErrors &&
+        !(
+            user.user_name.value === '' ||
+            (isNewUser &&
+                (((!user.password.value || user.password.value === '') &&
+                    !user.send_email_invitation.value) ||
+                    (user.send_email_invitation.value &&
+                        user.email?.value === '')))
```

Comment:
I don't understand this one: we need invitation to be false and have no email?

Reply:
this is a tricky one indeed, you have a ! before all this : 
```
!(
    user.user_name.value === '' ||
    (isNewUser &&
        (((!user.password.value || user.password.value === '') &&
            !user.send_email_invitation.value) ||
            (user.send_email_invitation.value &&
                user.email?.value === '')))
);
```